.travis.yml

```diff
@@ -9,12 +9,12 @@ services:
 
 script:
 # Do sanity checks
-- make sanity
+- env GO111MODULE=on make sanity
```

Comment:
If you bump to go `1.13.x` I think go modules are the default there (don't need the environment variable). If you don't want to move to go `1.13.x` you can either:

just add an environment section to the travis.yml

```
env:
    - GO111MODULE="on"
```

or make the first line of the script `export GO111MODULE="on"`.